components/core/SlateMediaObject.js

```diff
@@ -50,7 +50,8 @@ const STYLES_IMAGE = css`
 export default class SlateMediaObject extends React.Component {
   render() {
     const name = `${this.props.data.name}`;
-    const url = this.props.data.url ? this.props.data.url : `https://hub.textile.io${this.props.data.ipfs}`;
+    const cid = this.props.data.ipfs.replace('/ipfs/', '')
```

Comment:
a later enhancement might be to just track the CIDs in all data models and avoid the ipfs path or urls.